ergotree-interpreter/src/eval/sglobal.rs

```diff
@@ -486,11 +535,41 @@ mod tests {
         );
     }
 
+    #[test]
+    fn deserialize_group_element() {
```

Comment:
makes sense to test more types

Reply:
Roundtrip deserialization for types is done in serialize_roundtrip test